kubernetes/cray-bos/templates/db.yaml

```
@@ -55,14 +55,21 @@ spec:
       labels:
         app.kubernetes.io/name: "{{ include "cray-service.name" . }}-db"
     spec:
+      securityContext:
+        fsGroup: 65534
       containers:
       - name: "{{ include "cray-service.name" . }}-db"
         image: "{{ .Values.database.image.repository }}:{{ .Values.database.image.tag }}"
-        command: ["/bin/sh"]
-        args: ["-c", "echo 'save 10 1' >> /etc/redis.conf && sleep 1 && redis-server /etc/redis.conf"]
+        securityContext:
+          runAsUser: 65534
+          runAsGroup: 65534
         volumeMounts:
         - mountPath: /data
           name: "{{ include "cray-service.name" . }}-db"
+        - name: redis-config
```

This matters less since it's local to this chart, but for the database, we use the service prefix. So to be consistent, perhaps consider
```suggestion
        - name: "{{ include "cray-service.name" . }}-redis-config"
```

---

updated to be consistent.